util/object_pool.c

```diff
@@ -355,16 +375,9 @@ int comdb2_objpool_return(comdb2_objpool_t op, void *obj)
     return objpool_return_int(op, obj);
 }
 
-int comdb2_objpool_available(comdb2_objpool_t op)
+int comdb2_objpool_notify(comdb2_objpool_t op, int force)
 {
-    int avail;
-
-    Pthread_mutex_lock(&op->data_mutex);
-
-    avail = !exhausted(op) || !full(op);
-
-    Pthread_mutex_unlock(&op->data_mutex);
-    return avail;
+    return objpool_notify_int(op, force);
```

Comment:
If comdb2_objpool_notify is merely a call to objpool_notify_int, let's get rid of it.

Reply:
I assume you mean get rid of objpool_notify_int(), since the public function is called from another subsystem.